services/brig/src/Brig/ZAuth.hs

```diff
@@ -90,52 +103,68 @@ runZAuth :: MonadIO m => Env -> ZAuth a -> m a
 runZAuth e za = liftIO $ runReaderT (unZAuth za) e
 
 data Settings = Settings
-    { _keyIndex             :: !Int                   -- ^ Secret key index to use
-                                                      --   for token creation
-    , _userTokenTimeout     :: !UserTokenTimeout      -- ^ User token validity timeout
-    , _sessionTokenTimeout  :: !SessionTokenTimeout   -- ^ Session token validity timeout
-    , _accessTokenTimeout   :: !AccessTokenTimeout    -- ^ Access token validity timeout
-    , _providerTokenTimeout :: !ProviderTokenTimeout  -- ^ Access token validity timeout
+    { _keyIndex                    :: !Int                         -- ^ Secret key index to use
+                                                                   --   for token creation
+    , _userTokenTimeout            :: !UserTokenTimeout            -- ^ User token validity timeout
+    , _sessionTokenTimeout         :: !SessionTokenTimeout         -- ^ Session token validity timeout
+    , _accessTokenTimeout          :: !AccessTokenTimeout          -- ^ Access token validity timeout
+    , _providerTokenTimeout        :: !ProviderTokenTimeout        -- ^ Proider token validity timeout
+    , _legalHoldUserTokenTimeout   :: !LegalHoldUserTokenTimeout   -- ^ Legal Hold User token validity timeout
+    , _legalHoldAccessTokenTimeout :: !LegalHoldAccessTokenTimeout -- ^ Legal Hold Access token validity timeout
     } deriving (Show, Generic)
 
 defSettings :: Settings
 defSettings = Settings 1
-    (UserTokenTimeout (60 * 60 * 24 * 28))    -- 28 days
-    (SessionTokenTimeout (60 * 60 * 24))      -- 1 day
-    (AccessTokenTimeout 900)                  -- 15 minutes
-    (ProviderTokenTimeout (60 * 60 * 24 * 7)) -- 7 days
+    (UserTokenTimeout (60 * 60 * 24 * 28))          -- 28 days
+    (SessionTokenTimeout (60 * 60 * 24))            -- 1 day
+    (AccessTokenTimeout 900)                        -- 15 minutes
+    (ProviderTokenTimeout (60 * 60 * 24 * 7))       -- 7 days
+    (LegalHoldUserTokenTimeout (60 * 60 * 24 * 56)) -- 56 days
+    (LegalHoldAccessTokenTimeout (60 * 15 ))        -- 15 minutes
```

Comment:
```suggestion
    (LegalHoldAccessTokenTimeout (60 * 15))        -- 15 minutes
```

Reply:
(anyway i think @jschaul you said yourself a while ago that we shouldn't have defaults in the code, but enforce the existence of an exhaustive options file.  the more i think about it, the more i agree.  make a `FUTUREWORK` here about this?)

Reply:
That's not what I said, please don't misquote me :). There can be defaults in the code as long as those defaults can be overridden via configuration files. The discussion related to a hardcoded setting "by default SSO is disabled and must be enabled per-team via backoffice". And there was no way to change this behaviour via configuration - but now there is, thanks to your work on that other PR). This part of the code is different, as any default value here can easily be overridden in configuration, if, for some installation, expiry times should behave differently. In fact, we make use of that in two separate places at the moment.